src/components/CurrencyInputPanel/index.tsx

```diff
@@ -4,6 +4,7 @@ import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
 import { Pair } from '@uniswap/v2-sdk'
 import { useWeb3React } from '@web3-react/core'
 import { TraceEvent } from 'analytics'
+import PrefetchBalancesWrapper from 'components/AccountDrawer/PrefetchBalancesWrapper'
```

Comment:
path name might lead people to believe this is exclusively an account component--should prob bump it up to a root level component now that it's used here